### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by kal-aalrajhi**
## Description 
This feature has a fully built out user dashboard that welcomes the user by name, displays the current date and allows users to interact with all bookings. Past, today, and future bookings can be displayed at the click of a button. If a specific booking isn't available, a user is given an apologetic message letting them know that a *type* of booking isn't available. Moreover the total cost of a set of bookings is displayed and changes based off whether they are viewing **all bookings**, **past bookings**, **todays bookings** or **future bookings**.

#### How to test?
To run the server used, locally, please clone and [follow the instructions from this repo](#https://github.com/turingschool-examples/overlook-api).

run `npm start`

from there, click the *dashboard* button and view the dashboard. You can test features by clicking around or open up the *dev tools* to explore what certain elements are doing.

#### Screenshots 
Outlook Hotel Dashboard Sample

#### Type of Change:
- [x] New feature (non-breaking change which adds functionality)

#### How Has This Been Tested?

What steps have you taken to ensure the code is effective?
- [x] Have you used Dev Tools/ ran code in the console?
- [x] Have you verified changes using `open index.html`?

#### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] My commit messages are consistent, descriptive, concise and begin with a verb and capital letter.
- [x] I have performed a self-review of my own code.
- [x] I have added reviewers.

### CODE REVIEW

File: src/classes/User.js [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
     }
 
-    // Test w/ sample data
-    removeBooking(booking) {
-        // can only remove bookings from current to future dates
-        this.allBookings.splice() // find the booking to splice
-    }
+    // // Test w/ sample data
+    // removeBooking(booking) {
+    //     // can only remove bookings from current to future dates
```
Comment (aominhlong) [inline src/classes/User.js:19]: For now I think keeping your notes are ok but definitely remove them later

File: src/scripts.js [hunk 2]
```diff
@@ -34,13 +34,30 @@ const head = document.querySelector("#head");
 const subHead = document.querySelector("#subHead");
 const footer = document.querySelector("#footer");
 
+const bookingHistoryOptions = document.querySelector("#bookingHistoryOptions");
+
 // Event Listeners 
 window.addEventListener("load", () => loadData());
```
Comment (aominhlong) [inline src/scripts.js:41]: spacing

File: src/scripts.js [hunk 3]
```diff
@@ -60,22 +77,26 @@ const loadData = () => {
...
         tempData.forEach(bookingData => {
             allBookingsData.push(new Booking(bookingData));
         });
-        // console.log("booking", allBookingsData);
         
         tempData = data[2].rooms; 
         tempData.forEach(roomData => {
             allRoomsData.push(new Room(roomData));
         });
         // console.log("rooms", allRoomsData);
```
Comment (aominhlong) [inline src/scripts.js:91]: remove console logs when you're done

**Comment by aominhlong** [comment]
This looks super good! The comments I made above apply for all files. Good work so far!

**Review by kal-aalrajhi — COMMENTED** [review]
Awesome! Thank you for the helpful feedback!

**Pull request merged**